content/pt-br/horizontal-scaling.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Escalonamento Horizontal
+status: Completed
+category: Conceito
+tags: ["infrastructure", "", ""]
```

Comment:
You forgot the tag.

Reply:
"infraestrutura" instead "infrastructure"

Like Edson said, "just a tiny fix". After that LGTM!

Thank you!